composer.json

```diff
@@ -20,14 +20,17 @@
     ],
     "require": {
         "php": "^5.6 || ^7.0",
+        "php-http/httplug": "^1.1",
```

Comment:
This still needs to be moved to require-dev as not everyone uses this specific validator. And optionally to prevent getting unsupported versions a conflict rule, but this mainly depends on if `php-http/httplug` has a version lower then 1.0 which is still actively used today.

Reply:
There is only a v0.1.0 then 1.0.x releases. I'm pretty sure httplug came about because of PSR7 and dependency issues with bundles requiring different versions of guzzle. So 1.0 was the first real version

Reply:
OK, no need for a conflict rule then 👍